package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "openfin-workspace-starter",
+  "version": "1.0.0",
+  "description": "Getting Started with the OpenFin Workspace",
+  "scripts": {
+    "start": "openfin -l -c https://getting-started.openfin.co/app.json"
```

Comment:
I think this is an awesome idea, but it might not make sense to include the script until we have this. I would rather this be split into a separate PR where we actually add the thing. 100% on board with the concept though.

Reply:
Ah yes that's a good call. Will split this out

Reply:
Removed this from this PR, have a new WIP PR here to figure it out https://github.com/openfin/workspace-starter/pull/4